holoviews/plotting/bokeh/element.py

```diff
@@ -512,7 +552,7 @@ def initialize_plot(self, ranges=None, plot=None, plots=None, source=None):
         self.handles['plot'] = plot
 
         # Get data and initialize data source
-        empty = self.callbacks and self.callbacks.downsample
+        empty = False
         if self.batched:
```

Comment:
Yes, I'm going to end up getting rid of empty everywhere I think. But probably in a separate PR.